docs/sources/reference/run.md

```diff
@@ -882,9 +882,24 @@ command is not available for this logging driver
 
 Journald logging driver for Docker. Writes log messages to journald; the container id will be stored in the journal's `CONTAINER_ID` field. `docker logs` command is not available for this logging driver.  For detailed information on working with this logging driver, see [the journald logging driver](reference/logging/journald) reference documentation.
 
-#### Log Opts : 
+#### Logging driver: fluentd
 
-Logging options for configuring a log driver. The following log options are supported: [none]
+Fluentd logging driver for Docker. Writes log messages to fluentd (forward input). `docker logs`
+command is not available for this logging driver
+
+Some options are supported by specifying `--log-opt` as many as needed, like `--log-opt host=localhost --log-opt port=24224`.
```

Comment:
isn't this section wrong now? there is only `address` log opt (and the others about tag) so `host` and `port` are not gone right?